python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-ssh.j2

```diff
@@ -8,7 +8,8 @@
 !
 management ssh
 {%     if management_ssh.access_groups is arista.avd.defined %}
-{%         for access_group in management_ssh.access_groups %}
+{# TODO mark name as key required in the schema #}
```

Comment:
this template you already covered in part2

Reply:
Removed.